<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EDRTitleBar</class>
 <widget class="QWidget" name="EDRTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>26</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>26</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>26</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>EDRTitleBar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EDRTitleBar {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
 }

QToolButton { /* all types of tool button */
	border: 2px solid #8f8f91;
	border-radius: 6px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1 stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QToolButton[popupMode=1] { /* only for MenuButtonPopup */
	padding-right: 20px; /* make way for the popup button */
}

QToolButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1 stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* the subcontrols below are used only in the MenuButtonPopup mode */
QToolButton::menu-button {
	border: 2px solid gray;
	border-top-right-radius: 6px;
	border-bottom-right-radius: 6px;
	/* 16px width + 4px for border = 20px allocated above */
	width: 16px;
}

QToolButton::menu-arrow {
	image: url(downarrow.png);
}

QToolButton::menu-arrow:open {
	top: 1px; left: 1px; /* shift it a bit */
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="leftFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="rightFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonFloat">
     <property name="text">
      <string>o</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonClose">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/resources/icons/oddokowywanie.png</normaloff>
     <normalon>:/resources/icons/dokowanie.png</normalon>:/resources/icons/oddokowywanie.png</iconset>
   </property>
   <property name="text">
    <string>Float</string>
   </property>
   <property name="toolTip">
    <string>Float</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/resources/icons/x.png</normaloff>:/resources/icons/x.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ConsoleWidget.qrc"/>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>setWordWrap(bool)</slot>
 </slots>
</ui>
